patches/components-vector_icons-aggregate_vector_icons.py.patch

```diff
@@ -1,7 +1,16 @@
 diff --git a/components/vector_icons/aggregate_vector_icons.py b/components/vector_icons/aggregate_vector_icons.py
-index 3e5a94a7f053ffc66f1a820073265cbfe1332b39..0175ec714b441df5e8fa58e9391e2de6bbd67755 100644
+index 3e5a94a7f053f..505f1598a7216 100644
 --- a/components/vector_icons/aggregate_vector_icons.py
 +++ b/components/vector_icons/aggregate_vector_icons.py
+@@ -23,7 +23,7 @@ def Error(msg):
+ 
+ 
+ def CamelCase(name, suffix):
+-  words = name.split("_")
++  import re; words = re.split("_|-", name)
```

Comment:
This is required because a (very) large number of Leo Icons contain a `-` which this script did not deal with well.